lib/buildScripts.coffee

```diff
@@ -0,0 +1,28 @@
+spawn = require('child_process').spawn
+
+module.exports =
+
+    build: ->
+        # start Hint
+        atom.notifications.addInfo 'Build started', dismissable: true, detail: "Please don't start Arma until the build is completed."
```

Comment:
Better might be: "Please don't launch Arma" or "Please don't load the project folder" ?